Scripts/Restriction/ranacompiler.py

```diff
@@ -102,20 +104,12 @@ def regex(site):
 
 
 def is_palindrom(sequence):
-    """is_palindrom(sequence) -> bool.
-
-    True is the sequence is a palindrom.
-    sequence is a Seq object.
-    """
+    """Is the sequence (Seq objecT) as a palindrom."""
```

Comment:
I feel this is better phrased as "Whether the sequence is a palindrome or not." (also there is a typo in the end currently)